### PR TITLE
Checklock: Read AWS env variables

### PR DESCRIPTION
**Pull request by KathiaBarahona**
For output comparison we read and write into AWS bucket, this PR just includes enabling aws env variables in checklock

### CODE REVIEW

File: modelhub/tests_modelhub/checklock_holmes/.env_sample [hunk 1]
```diff
@@ -17,3 +17,9 @@ METABASE_PASSWORD = "metabase1"
 METABASE_DASHBOARD_ID = '1' # 1-model-hub
 METABASE_DATABASE_ID = '2' # objectiv database
 METABASE_COLLECTION_ID = '2' # 2-object
+
+# AWS
```
Comment (jansentom) [inline modelhub/tests_modelhub/checklock_holmes/.env_sample:21]: Bit confusing with athena above? What about making it slightly more descriptive,
REPORT__AWS_BUCKET_NAME etc maybe?

**Review by jansentom — APPROVED** [review]
Just a comment on naming. LGTM otherwise!

File: modelhub/tests_modelhub/checklock_holmes/.env_sample (hunk 1 above)
Comment (KathiaBarahona) [reply modelhub/tests_modelhub/checklock_holmes/.env_sample:21]: true true

Comment (KathiaBarahona) [reply modelhub/tests_modelhub/checklock_holmes/.env_sample:21]: renamed here: https://github.com/objectiv/objectiv-analytics/pull/1403/commits/c85e55b18c85b4e02de3fb990835120ab0c77f36

**Pull request merged**